README.md

```diff
@@ -72,10 +72,44 @@ $ mix torch.gen.html Blog Post posts title:string body:text published_at:datetim
 The output would look like:
 
 ```bash
-Add the resource to your browser scope in lib/my_app_web/router.ex:
+* creating priv/templates/phx.gen.html/edit.html.heex
+* creating priv/templates/phx.gen.html/form.html.heex
+* creating priv/templates/phx.gen.html/index.html.heex
+* creating priv/templates/phx.gen.html/new.html.heex
+* creating priv/templates/phx.gen.html/show.html.heex
+* creating priv/templates/phx.gen.html/controller_test.exs
+* creating priv/templates/phx.gen.html/controller.ex
+* creating priv/templates/phx.gen.html/view.ex
+* creating priv/templates/phx.gen.context/access_no_schema.ex
+* creating priv/templates/phx.gen.context/context.ex
+* creating priv/templates/phx.gen.context/schema_access.ex
+* creating priv/templates/phx.gen.context/test_cases.exs
+* creating priv/templates/phx.gen.context/context_test.exs
+* creating lib/phx1_6_web/controllers/post_controller.ex
+* creating lib/phx1_6_web/templates/post/edit.html.heex
+* creating lib/phx1_6_web/templates/post/form.html.heex
+* creating lib/phx1_6_web/templates/post/index.html.heex
+* creating lib/phx1_6_web/templates/post/new.html.heex
+* creating lib/phx1_6_web/templates/post/show.html.heex
+* creating lib/phx1_6_web/views/post_view.ex
+* creating test/phx1_6_web/controllers/post_controller_test.exs
+* creating lib/phx1_6/blog/post.ex
+* creating priv/repo/migrations/20220201154756_create_posts.exs
```

Comment:
`mix torch.gen.htm` creates a lot of files, besides templates, that seems like they are pretty important. Might be worth highlighting some of them.

Reply:
Now that I see this, I'm not sure I want to keep this list in the README.  This list of files will surely change over time as Phoenix changes, and I don't want us to have to make sure it's always exhaustive and correct.   

If we do anything what about just showing the first line or two, then ... then the final line or two just to simulate what to expect?

```bash
* creating priv/templates/phx.gen.html/edit.html.heex
* creating priv/templates/phx.gen.html/form.html.heex
...<omitted for brevity>...
* injecting test/phx1_6/blog_test.exs
* injecting test/support/fixtures/blog_fixtures.ex
```